### ISSUE TITLE
[Backend] Setup CD pipelines

### ISSUE THREAD

**Issue by markgravity** [markgravity/flutter-survey#32]
## Why

No manual execution of releases

## Acceptance Criteria
Auto build and upload a build to AppStore & Google Play when release a version

**Comment by sleepylee** [comment]
this one is a chore...when we work on a real project it will be counted so fyi 😄

**Comment by markgravity** [comment]
> this one is a chore...when we work on a real project it will be counted so fyi 😄

Don't get your point here, you mean the title or the label? 🤔

**Comment by sleepylee** [comment]
yes - so it's not considered as a Backend thing 😅 you can use prefix [Chore] which is more precise..
